src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -98,6 +102,29 @@ public void setPersons(List<Person> persons) {
         internalList.setAll(persons);
     }
 
+    /**
+     * Updates the tag in every contact.
+     * @param oldTag The old tag to be changed.
+     * @param newTag The new tag that is changed into.
+     */
```

Comment:
There seems to be a missing empty line and the params should not have periods `.` at the end.

Reply:
OK . I made the changes.